test/e2e/metrics_test.go

```diff
@@ -287,3 +289,24 @@ func (tc *testContext) ensureMonitoringIsEnabled() error {
 
 	return nil
 }
+
+// testNodeResourceUsage ensures cpu/memory usage information is retrievable from Windows nodes
+func testNodeResourceUsage(t *testing.T) {
+	_, err := NewTestContext()
+	require.NoError(t, err)
+
+	// Need at least one Windows node to run these tests, throwing error if this condition is not met
+	require.Greater(t, len(gc.nodes), 0, "test requires at least one Windows node to run")
+
+	for _, winNode := range gc.nodes {
+		t.Run(winNode.Name, func(t *testing.T) {
+			cmd := exec.Command("oc", "adm", "top", "node", winNode.Name)
```

Comment:
Should we make this an API call?

Reply:
This may be the way forward if we want to avoid 
> test very k8s feature out there in our e2e tests

Updated this test case in [a new PR](https://github.com/openshift/windows-machine-config-operator/pull/453). Please let me know if it is better.

Reply:
Please remove this commit from this PR so that we can get it merged.

Reply:
On second thought, the test may be robust if we also included this logic/asserts as `oc adm top node` checks current resource usage stats and the API call checks allocatable resource stats. Both are useful to be checked in my opinion. Thoughts?

Reply:
Like I said before, please remove this commit from this PR and move this discussion to #453